eng/pipelines/smoke-tests.yml

```diff
@@ -11,34 +11,69 @@ jobs:
           OSVmImage: "ubuntu-16.04"
           TestTargetFramework: netcoreapp2.1
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         Windows_NetCoreApp (Public):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: netcoreapp2.1
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         Windows_NetFramework (Public):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: net461
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         MacOs (Public):
           OSName: "MacOS"
           OSVmImage: "macOS-10.15"
           TestTargetFramework: netcoreapp2.1
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         Windows_NetCoreApp (Gov):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: netcoreapp2.1
           CloudType: gov
+          AdditionalParameters: $(govAdditionalParameters)
+          Location: $(govLocation)
         Windows_NetFramework (Gov):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: net461
           CloudType: gov
+          AdditionalParameters: $(govAdditionalParameters)
+          Location: $(govLocation)
+        Windows_NetCoreApp (China):
+          OSName: "Windows"
+          OSVmImage: "windows-2019"
+          TestTargetFramework: netcoreapp2.1
+          CloudType: cn
+          AdditionalParameters: $(cnAdditionalParameters)
+          Location: $(cnLocation)
+        Windows_NetFramework (China):
+          OSName: "Windows"
+          OSVmImage: "windows-2019"
+          TestTargetFramework: net461
+          CloudType: cn
+          AdditionalParameters: $(cnAdditionalParameters)
+          Location: $(cnLocation)
+
     pool:
       vmImage: $(OSVmImage)
 
+    variables:
+      publicAdditionalParameters: "@{ storageEndpointSuffix = 'core.windows.net'; authorityHost = 'https://login.microsoftonline.com'; location = '$(publicLocation)' }"
```

Comment:
If we expect these to be the same for all public and gov cloud types then I'd suggest moving them into the eng/common templates and if needed possibly allow for overriding.